packages/mui-base/src/TextareaAutosize/TextareaAutosize.tsx

```diff
@@ -197,6 +197,7 @@ const TextareaAutosize = React.forwardRef(function TextareaAutosize(
         ref={handleRef}
         // Apply the rows prop to get a "correct" first SSR paint
         rows={minRows as number}
+        style={style}
```

Comment:
As `style` is not used besides this line, we could instead remove it from the destructure in line 63.

Reply:
@DiegoAndai It is used on line 211 on the hidden textarea.

Reply:
My bad, I only checked upwards 😅

I'm not familiar with this component implementation. Why are there two `textarea`s? Is it expected that `style` would be applied to both?

Reply:
I initially had some confusion about this, but here's my reasoning:

By default, `<input>` and `<textarea>` elements don’t change size based on the content they contain. 

The second textarea which is hidden from view, is used for calculating the height based on the content. This textarea adjusts its height dynamically as the user types or modifies the content.

The first textarea is the one that users interact with directly. It mirrors the content of the hidden textarea and adjusts its height accordingly, ensuring that the visible textarea always matches the calculated height.

I also came across these articles:

- https://css-tricks.com/auto-growing-inputs-textareas/

- https://css-tricks.com/the-cleanest-trick-for-autogrowing-textareas/

So, I believe the `style` should be applied to both textareas since one is an exact replica of the other.

Reply:
That makes sense, thanks for the detailed explanation